src/test/java/uk/gov/companieshouse/api/accounts/transformer/IntangibleAssetsTransformerTest.java

```diff
@@ -154,7 +162,7 @@ void TestGoodwillMapFromRestObjectCostAndAmortisationNull() {
     }
 
     @Test
-    @DisplayName("Test other intangible assets field map from the REST object when the nested cost objects is not null")
+    @DisplayName("Test other intangible assets assets field map from the REST object when the nested cost objects is not null")
```

Comment:
Is this correct?

Reply:
No, thanks for catching it, i'll correct it now and double check the others